### ISSUE TITLE
Underlying synapse-tools changes will break these workflows

### ISSUE THREAD

**Issue by allaway** [Sage-Bionetworks/rare-disease-workflows#13]
See note from @thomasyu888:

> I am taking over as maintainer for: https://github.com/Sage-Bionetworks/synapse-client-cwl-tools.  Part of this work will be configuring this repository to mimic that of the new dockstore-tool-* format.  There will be some changes so PLEASE make sure you are using the v0.1 tag: https://github.com/Sage-Bionetworks/synapse-client-cwl-tools/tree/v0.1.  Changes:
* The repository will be renamed to Sage-Bionetworks-Workflows/dockstore-tool-synapse  (Github redirects old links, so as long as you are using v0.1 tag, your tools that reference the tools here will be fine)
* The tools will be moved into a cwl folder (so the next version you use, you will have to update the path to the workflow tools)
* Tests will be added to each of the tools

Basically, we'll need to update the tools to reference the v0.1 tag if we want to use them in the future. 

cc @jaybee84

**Comment by jaybee84** [comment]
ack

**Comment by thomasyu888** [comment]
@allaway + @jaybee84 . It is good practice to always use specific tagged releases of workflows, because the github branches can change and be updated but the tagged content cannot.  I fixed for you guys in linked PR.
